### DISCUSSION TITLE
StrongSwan VPPN to Azure is unstable, the channel goes down - no errors?

### DISCUSSION THREAD

**Discussion by ARycr0ft** [strongswan/strongswan#2241] [Q&A]
Hi,

Please can you advise me? We are trying to use the StrongSwan software to manage our VPNs. We are trying to configure a VPN into Azure. For some reason this channel keeps on going down. There are no error messages, and we do not know where to start looking. If anybody can provide some guidance it would be appreciated?

We tried configuring the on-premise IPsec tunnel to ResponseOnly (auto=add) and Azure to InitiatorOnly, but this did not work.

We rolled the on-premise IPsec back to InitiatorOnly (auto=start), and changed the Azure connection to “Default”, which implies that Azure is both Responder and Initiator. The channel did come back up, but dropped again after a few days. The log files do not indicate any errors.

Any tips on how to diagnose and isolate the problem would be appreciated.

Kind regards
Andrew Rycroft

**Comment by tobiasbrunner** [comment]
> For some reason this channel keeps on going down. There are no error messages, and we do not know where to start looking.

Start by reading the logs. There has to be some reason it "goes down" as you say. Or what do you mean with "goes down" exactly?

> We tried configuring the on-premise IPsec tunnel to ResponseOnly (auto=add) and Azure to InitiatorOnly, but this did not work.

Why not? What doesn't work?

Anyway, you might want to consider using traffic triggered initiation (`auto=route`) so the SA is automatically recreated when traffic matches.
